connect_ext_ppr/validator.py

```diff
@@ -84,16 +85,18 @@ def validate_pricelist_ids(client, product_id, req_mp_configs, dep_mp_configs):
         })
 
 
-def validate_marketplaces_ppr(ppr, dr_marketplaces, dep_marketplaces):
+def validate_marketplaces_ppr(ppr, dr_marketplaces, dep_marketplaces, db):
     """
     Validates that we can apply the ppr to all marketplaces
     """
     dep_mkplc_map = {m.marketplace: m for m in dep_marketplaces}
     mkplcs_w_erros = []
     for mp_data in dr_marketplaces:
         mp_id = mp_data.id
-        if dep_mkplc_map[mp_id].ppr_id and dep_mkplc_map[mp_id].ppr_id > ppr.id:
```

Comment:
@Hairash I think that if you only change `dep_mkplc_map[mp_id].ppr_id > ppr.id` to `dep_mkplc_map[mp_id].ppr.version > ppr.version` it works. The test `tests/api/test_deployment_requests.py::test_create_deployment_invalid_ppr_for_marketplace` continue passing with this change.

Reply:
Really, I missed that ppr is relationship for MarketplaceConfiguration. Thanks, Jonathan!

Reply:
Also changed test to be sure it works for case mp's ppr_id greater than request's ppr.id